src/components/attendee/VideoComms/internal/scss/TwilioTrackDisplay.module.scss

```diff
@@ -0,0 +1,3 @@
+.mirrored {
```

Comment:
no BEM?

Reply:
I don't think it's necessary when we're using styled components. Just looks like more noise to me.

Reply:
I'd still say that it improves the overall project navigation. When you see .mirrored-${id}, you might still have multiple occurrences of this class. Making the naming consistent helps to avoid such situations

Reply:
@mike-lvov @colinhowe I have shared some good example here that I've noticed a 3rd party library has used and it was helpful for us:
https://doesitsparkle.slack.com/archives/C01ER4M61TL/p1644404537802149?thread_ts=1644335099.082349&cid=C01ER4M61TL

@colinhowe semantical meaning can be added in other ways just as well. we've not touched `data-*` props much.
I guess people in React avoid them, not necessarily only because they don't have camel casing.
The BEM approach is more versatile than just project navigation. It makes it easy to report issues with precise data and makes it easy to test solutions by just the browser dev tools.

Reply:
I think CSS source maps make it easy to find where class is defined when you're debugging things.

That said, I don't feel particularly strongly about this.

`TwilioTrackDisplay__mirrored`?

Reply:
That should be a modifier, no?

Reply:
`TwilioTrackDisplay--mirrored`